### PR TITLE
Move repos config into sqlite

### LINKED ISSUE #139: Support concurrent access to admin web UI
The admin web UI is not very friendly to multiple concurrent users. Easy to have people stepping on each others toes. Would be better to add/remove entries one at a time rather than doing a wholesale replace each time.

### PR DESCRIPTION
**Pull request by matthauck**
This finishes up what was started by #157.

Fixes #139, #153

### CODE REVIEW

File: src/server/octobot_service.rs [hunk 1]
```diff
@@ -83,8 +83,11 @@ impl OctobotService {
                     (&Post, "/api/users") => UserAdmin::new(self.config.clone(), Op::Create),
                     (&Delete, "/api/user") => UserAdmin::new(self.config.clone(), Op::Delete),
 
-                    (&Get, "/api/repos") => admin::GetRepos::new(self.config.clone()),
-                    (&Post, "/api/repos") => admin::UpdateRepos::new(self.config.clone()),
+                    (&Get, "/api/repos") => RepoAdmin::new(self.config.clone(), Op::List),
+                    (&Put, "/api/repo") => RepoAdmin::new(self.config.clone(), Op::Update),
+                    (&Post, "/api/repos") => RepoAdmin::new(self.config.clone(), Op::Create),
+                    (&Delete, "/api/repo") => RepoAdmin::new(self.config.clone(), Op::Delete),
```
Comment (connyay) [inline src/server/octobot_service.rs]: rest bikeshed - feel free to ignore:

I would have expected these routes to be:
```
POST    /api/repos      # Create
GET     /api/repos      # Read
PUT     /api/repos/:id  # Update
DELETE  /api/repos/:id  # Delete
```

Comment (matthauck) [reply src/server/octobot_service.rs]: Yeah, I definitely feel bad about these hacky-not-restful-standard routes, but I didn't want to get into URL parsing quite yet -- much easier to match the whole URL. And `/api/repo` seemed better for updates than `/api/repos` which looks more like a collection thing w/o an id on it. Will revisit!

**Comment by matthauck** [comment]
I was kinda sad in the end about making the `RepoInfo` properties no longer nullable to make the inserts/updates easier. This makes db interaction easier and some code cleaner, but is kinda less correct and there might be empty strings to check for instead of `None` to check for...? /shrug

**Comment by matthauck** [comment]
Made an update. Please take a look.

**Comment by matthauck** [comment]
Yeah -- I didn't make any db changes, just JSON deserialization changes for what do do when that element is null, i.e. _require_ it to be present for the `bool` values rather than choosing a default. 

I think I convinced myself on the drive in today that not null is actually fine and we can keep the schema as is. We have to choose a default and handle things like empty strings anyway, so might as well make them not null and keep the db code simpler.

**Pull request merged**